### PR TITLE
triggerpertimeframe

### PR DESCRIPTION
**Pull request by zarinabanoo**
added-trd-triggerpertimeframe-info

### CODE REVIEW

File: Modules/TRD/src/TrackletsTask.cxx [hunk 1]
```diff
@@ -119,9 +119,9 @@ void TrackletsTask::retrieveCCDBSettings()
   if (mChamberStatus == nullptr) {
     ILOG(Info, Support) << "mChamberStatus is null, no chamber status to display" << ENDM;
   }
-  //j  else{
-  //   drawHashedOnHistsPerLayer();
-  //  }
+  // j  else{
+  //    drawHashedOnHistsPerLayer();
+  //   }
```
Comment (tdietel) [inline Modules/TRD/src/TrackletsTask.cxx:124]: Why was this modified? Or why is this still in the code, if it is commented out?

File: Modules/TRD/src/TrackletsTask.cxx [hunk 2]
```diff
@@ -172,14 +172,19 @@ void TrackletsTask::buildHistograms()
   mTrackletsPerTimeFrameCycled.reset(new TH1F("trackletspertimeframecycled", "Number of Tracklets per timeframe, this cycle;Tracklets in TimeFrame;Counts", 25000, 0, 500000));
   getObjectsManager()->startPublishing(mTrackletsPerTimeFrameCycled.get());
 
+  //=====================newly added lines=================
```
Comment (tdietel) [inline Modules/TRD/src/TrackletsTask.cxx:175]: This comment is too generic. Remove it or be more specific.

File: Modules/TRD/src/TrackletsTask.cxx [hunk 3]
```diff
@@ -189,16 +194,16 @@ void TrackletsTask::drawHashOnLayers(int layer, int hcid, int rowstart, int rowe
...
+  boxlines[4] = new TLine(bottomleft.first, topright.second - (topright.second - bottomleft.second) / 2, topright.first, topright.second - (topright.second - bottomleft.second) / 2);     // horizontal middle
+  boxlines[5] = new TLine(topright.first, bottomleft.second, bottomleft.first, topright.second);                                                                                           // backslash
+  boxlines[6] = new TLine(bottomleft.first, bottomleft.second, topright.first, topright.second);                                                                                           // forwardslash
+  boxlines[7] = new TLine(bottomleft.first + (topright.first - bottomleft.first) / 2, bottomleft.second, bottomleft.first + (topright.first - bottomleft.first) / 2, topright.second);     // vertical middle
+  boxlines[8] = new TLine(bottomleft.first, bottomleft.second + (topright.second - bottomleft.second) / 2, topright.first, bottomleft.second + (topright.second - bottomleft.second) / 2); // bottom
```
Comment (tdietel) [inline Modules/TRD/src/TrackletsTask.cxx:206]: There are a lot of "false positives", where single spaces in a comment show up as changes. Can you undo these changes, or put them into a different PR if they are required/recommended due to some style guidelines.

**Review by tdietel — CHANGES_REQUESTED** [review]
I think this PR should still be cleaned up a bit. Please do not merge yet.

File: Modules/TRD/src/TrackletsTask.cxx (hunk 1 above)
Comment (zarinabanoo) [reply Modules/TRD/src/TrackletsTask.cxx:124]: Ok @tdietel ....I will remove all these...Thanks

File: Modules/TRD/src/TrackletsTask.cxx (hunk 2 above)
Comment (zarinabanoo) [reply Modules/TRD/src/TrackletsTask.cxx:175]: Ok....

File: Modules/TRD/src/TrackletsTask.cxx (hunk 3 above)
Comment (zarinabanoo) [reply Modules/TRD/src/TrackletsTask.cxx:206]: @tdietel ...It is due to Clang format
